src/native/external/zlib-ng/CMakeLists.txt

```diff
@@ -462,29 +462,31 @@ endif()
 #
 # check for ptrdiff_t support
 #
-check_c_source_compiles(
-    "#include <stddef.h>
-     int main() {
-         ptrdiff_t *a;
-         (void)a;
-         return 0;
-    }"
-    HAVE_PTRDIFF_T
-)
-if(NOT HAVE_PTRDIFF_T)
-    set(NEED_PTRDIFF_T 1)
-
-    check_type_size("void *" SIZEOF_DATA_PTR)
-    message(STATUS "sizeof(void *) is ${SIZEOF_DATA_PTR} bytes")
 
-    if(${SIZEOF_DATA_PTR} MATCHES "4")
-        set(PTRDIFF_TYPE "uint32_t")
-    elseif(${SIZEOF_DATA_PTR} MATCHES "8")
-        set(PTRDIFF_TYPE "uint64_t")
-    else()
-        message(FATAL_ERROR "sizeof(void *) is neither 32 nor 64 bit")
-    endif()
-endif()
+# TODO: SIZEOF_DATA_PTR is undefined when using MSVC because /INCREMENTAL:NO keeps getting overwritten to /INCREMENTAL by CMake.
```

Comment:
Workaround as we were unable to override the /INCREMENTAL value, as it was getting overriden by MSVC.

Reply:
Resolving as I pushed a fix.

Reply:
Thanks @jkoritzinsky for fixing this in https://github.com/dotnet/runtime/pull/102403/commits/2198dc38d6fa004a31adabc346001787511bb8cf